freqtrade/leverage/__init__.py

```diff
@@ -1 +1,3 @@
 # flake8: noqa: F401
+from freqtrade.leverage.get_interest import get_interest
```

Comment:
I'd call the file `interest.py` instead of `get_interrest`.

That'll make the imports sound "nicer" - as "get-interest" is an action / method name - but it's not well suited as a module i'd say